docs/source/using/profiles.adoc

```diff
@@ -8,18 +8,45 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+Profiles share nothing between them. Things like cache, add-ons and persistent config are specific to a single profile.
+This means if you have set some specific configuration parameters for a profile, these settings do not apply to other profiles.
+
+Currently, we cannot set any global configuration for profiles.
+If you want to apply a common configuration to all your profiles then you need to apply the settings for each profile separately.
+As a workaround, xref:../using/basic-usage.adoc#environment-variables[environment variables] can be used.
+
+To set 8GB memory for all profiles on Linux/macOS, run:
+
+----
+$ export MINISHIFT_MEMORY=8G
+----
+
+You can have multiple profiles and can switch between them. However only one profile can be active at any time. 
+you do not need to delete an existing {project} instance to test a different configuration.
+
+By default {project} commands will run against the active profile.
+When a profile is started it becomes the active profile. You can change the active profile using xref:switching-profiles[`profile set`] command.
+
+The profile feature introduced a global `--profile` flag and xref:../command-ref/minishift_profile.adoc#[commands] for listing, deleting and setting the active profile.
+
+The `--profile` global flag is used with other {project} commands to run a command against a specific profile.
+
+NOTE: Commands are run against the default profile If you are not using `--profile` flag  and not set a different active profile.
+
+[[creating-profiles]]
 == Creating Profiles
 
 You can create a new instance of {project} with xref:../command-ref/minishift_start.adoc#[`minishift start --profile`] command.
 
-NOTE: The default profile is `minishift`. It will be present by-default and user does not need to create it specifically.
+NOTE: The default profile is *minishift*. It will be present by-default and user does not need to create it specifically.
```

Comment:
Remove the hyphen in "by-default", please. There's also no need to address a user (directly or indirectly):

"This profile will be present by default and does not need to be explicitly created."

Note: I used "explicitly" rather than "specifically" here because the creation of the *minishift* profile is implicit.